packages/apputils/src/domutils.ts

```diff
@@ -50,4 +50,22 @@ export namespace DOMUtils {
   export function createDomID(): string {
     return `id-${UUID.uuid4()}`;
   }
+
+  /**
+   * Check whether the active element descendant from given parent is editable.
+   * When checking active elements it includes elements in the open shadow DOM.
+   */
+  export function hasActiveEditableElement(
+    parent: Node | DocumentFragment,
+    root: ShadowRoot | Document = document
+  ): boolean {
+    const element = root.activeElement;
+    return !!(
```

Comment:
For JavaScript purist points:

```suggestion
    return Boolean(
```

Reply:
I was being facetious when I said "purist." I think it essentially comes down to a stylistic choice. But a lot of programmers just don't know about it. :)